### PR TITLE
15.0 mig website legal page

### PR DESCRIPTION
**Pull request by schout-it**

### CODE REVIEW

**Review by ivantodorovich — APPROVED** [review]
Code review, LG :+1:

File: website_legal_page/hooks.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from copy import deepcopy
+from xml.sax.saxutils import escape
+
+from lxml import etree as ElementTree
+
+from odoo import SUPERUSER_ID, api
+
+
+def _merge_views(env, xmlids):
```
Comment (ivantodorovich) [inline website_legal_page/hooks.py:11]: Any chance we can have some unit tests for this method?
If you don't have the time, don't worry about it..

File: website_legal_page/controllers/main.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Alexandre Díaz - Tecnativa
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import http
+from odoo.http import request
+
+
+class WebsiteLegalPage(http.Controller):
+    @http.route(["/shop/terms"], type="http", auth="public", website=True)
```
Comment (tarteo) [inline website_legal_page/controllers/main.py:9]: In 15.0 this is `/terms` not `/shop/terms`
image

**Review by tarteo — CHANGES_REQUESTED** [review]

**Comment by JrAdhoc** [comment]
Functional review :+1: 

PR name should be: [MIG] website_legal_page: Migration to 15.0

**Review by ioans73 — APPROVED** [review]
Functional review 👍🏻

File: website_legal_page/controllers/main.py (hunk 2 above)
Comment (pedrobaeza) [reply website_legal_page/controllers/main.py:9]: This seems to be pending.

**Pull request closed without merging**